src/test/java/edu/kis/powp/jobs2d/TestJobs2dPatterns.java

```diff
@@ -39,8 +52,14 @@ private static void setupDrivers(Application application) {
 		DriverFeature.addDriver("Logger Driver", loggerDriver);
 		DriverFeature.getDriverManager().setCurrentDriver(loggerDriver);
 
-		Job2dDriver testDriver = new MyAdapter();
-		DriverFeature.addDriver("Buggy Simulator", testDriver);
+		Job2dDriver testDriver = new BasicDriverAdapter (DrawerFeature.getDrawerController());
+		DriverFeature.addDriver("Basic Line", testDriver);
+
+        Job2dDriver testDriver2 = new LineDrawerAdapter( DrawerFeature.getDrawerController(), LineFactory.getSpecialLine());
+        DriverFeature.addDriver("Special Line", testDriver2);
```

Comment:
Original code used tabs instead of spaces. It is important to keep the original convention. It makes tracking actual changes more difficult.